backend/src/email/routes/email-transactional.routes.ts

```diff
@@ -32,6 +32,11 @@ export const InitEmailTransactionalRoute = (
       attachments: Joi.array().items(Joi.object().keys().required()),
     }),
   }
+  const getByIdValidator = {
+    [Segments.PARAMS]: Joi.object({
+      emailId: Joi.number().required(),
```

Comment:
same as above about being liberal in what we accept 🤔

Reply:
Joi is already automatically converting numerical strings to number